hcsshim.go

```diff
@@ -119,3 +162,17 @@ func convertAndFreeCoTaskMemString(buffer *uint16) string {
 	coTaskMemFree(unsafe.Pointer(buffer))
 	return str
 }
+
+func convertAndFreeCoTaskMemBytes(buffer *uint16) ([]byte, error) {
+	// TODO this requires an extra copy of the returned string data,
+	// it should be replaced with a more efficient way.
+	return []byte(convertAndFreeCoTaskMemString(buffer)), nil
```

Comment:
Did you forget to push?

From: Darren Stahl [mailto:notifications@github.com]
Sent: Friday, April 22, 2016 5:44 PM
To: Microsoft/hcsshim <hcsshim@noreply.github.com>
Cc: John Starks <John.Starks@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Microsoft/hcsshim] Implementing the new interface using the new HCS API - TP5 Only (#36)


In hcsshim.go<https://github.com/Microsoft/hcsshim/pull/36#discussion_r60819160>:

> @@ -119,3 +162,17 @@ func convertAndFreeCoTaskMemString(buffer *uint16) string {

>       coTaskMemFree(unsafe.Pointer(buffer))

>       return str

>  }

> +

> +func convertAndFreeCoTaskMemBytes(buffer *uint16) ([]byte, error) {

> +     // TODO this requires an extra copy of the returned string data,

> +     // it should be replaced with a more efficient way.

> +     return []byte(convertAndFreeCoTaskMemString(buffer)), nil

@jstarks<https://github.com/jstarks> PTAL at updated function

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/hcsshim/pull/36/files/5ed29853294143046b90fda3a4504ee0c2507549#r60819160>